collection-scripts/gather

```diff
@@ -82,5 +82,8 @@ oc adm inspect --dest-dir must-gather --rotated-pod-logs "${group_resources_text
 # Gather metallb logs
 /usr/bin/gather_metallb_logs
 
+# Gather NMState logs
+/usr/bin/gather_nmstate_logs
```

Comment:
```suggestion
# Gather NMState CRs
/usr/bin/gather_nmstate_crs
```

Reply:
It is not only about CRs (but not only about logs neither). WDYT about `gather_nmstate`?

Reply:
Updated